rpm/__init__.py

```diff
@@ -64,19 +66,27 @@ def get_sitepackages_and_suffixes(interpreter):
     # try platform-python first (it could be the only interpreter present on the system)
     interpreters = [
         "/usr/libexec/platform-python",
-        f"/usr/bin/python{majorver}",
         f"/usr/bin/python{majorver}.{minorver}",
+        f"/usr/bin/python{majorver}",
```

Comment:
Isn't the order change enough to solve your issue? Skipping breaks the ubi9 test case (3.9 rpm module with 3.11 interpreter).

Reply:
Yes, it is